fed/mosaic-application/src/main/java/org/eclipse/mosaic/fed/application/ambassador/simulation/AbstractSimulationUnit.java

```diff
@@ -99,7 +94,6 @@ public abstract class AbstractSimulationUnit implements EventProcessor, Operatin
 
     private final AdHocModule adhocModule;
     private final CellModule cellModule;
-    private final SensorModule sensorModule;
```

Comment:
I don't understand why the module now goes into `VehicleUnit`. So implementing it here was wrong in the first place?

Reply:
Yes, as long as we only support vehicles with the proposed sensor modules, it's sufficient to initialize the implementation in the `VehicleUnit` only. We can discuss if we already want RSUs or TrafficLights to be able be `Sensible`, but again, this more sounds like a feature than a re-design.